.changeset/red-months-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aonic-ui/pipelines": minor
```

Comment:
This might be a patch instead of minor change.

Reply:
Updated.